client/my-sites/marketing/index.node.js

```diff
@@ -0,0 +1,12 @@
+export default function ( router ) {
+	router( [ '/marketing/do-it-for-me', '/marketing/do-it-for-me/*' ], ( { res } ) => {
+		res.redirect( 301, 'https://wordpress.com/do-it-for-me/' );
+	} );
+
+	router(
+		[ '/marketing/ultimate-traffic-guide', '/marketing/ultimate-traffic-guide/*' ],
+		( { res } ) => {
+			res.redirect( 301, 'https://wpcourses.com/course/intro-to-search-engine-optimization-seo/' );
+		}
+	);
+}
```

Comment:
Currently, we'll only handle these couple of routes on the server-side, and the rest will never be passed down to be handled at the client level. In order to pass the rest of the routes handling down to the client side, you might need a catch-all route with a wildcard, something like:

```suggestion
import { makeLayout } from 'calypso/controller';

export default function ( router ) {
	router( [ '/marketing/do-it-for-me', '/marketing/do-it-for-me/*' ], ( { res } ) => {
		res.redirect( 301, 'https://wordpress.com/do-it-for-me/' );
	} );

	router(
		[ '/marketing/ultimate-traffic-guide', '/marketing/ultimate-traffic-guide/*' ],
		( { res } ) => {
			res.redirect( 301, 'https://wpcourses.com/course/intro-to-search-engine-optimization-seo/' );
		}
	);

	router( '/marketing/*', makeLayout );
	router( '/sharing/*', makeLayout );
}

```

Note that I'm including `/sharing` as well, since it's another top-level path that this section is handling on the client side.